highfidelity/hifi
# PAL: Admin Labels, Updated Letterboxes

This PR adds "ADMIN" labels to the right of Display Names on PAL cards corresponding to domain administrators. This PR also updates the visual style of the letterbox popups that appear when clicking on "[?]" buttons (and in some error cases).

**Test Plan**
This test plan requires two clients running this PR: Clients A and B. Run Sandbox from Client A; Client A will have admin privileges on the domain.
* On Clients A and B, join Sandbox A
* Open the PAL on both clients
* On Client A, verify that both Client A's and Client B's usernames appear below their display names in the PAL.
* On Client B, verify that only Client B's username appears below their display name in the PAL.
* On Client A, use the PAL to change its display name to something very long, like "TESTING TESTING TESTING TESTING".
* On Client B, refresh the PAL. Verify the presence of an "ADMIN [?]" label to the right of Client A's elided display name
* Change Client A's display name to something short, like "Test".
* On Client B, refresh the PAL. Verify that the "ADMIN [?]" label is still present to the right of Client A's un-elided display name.
* On Client A, verify that there is no label to the right of Client B's display name
* On Client B, click on the "[?]" button to the right of the "ADMIN" text. Verify that the popup appears and looks better than the old letterbox :smile:. Close the popup by clicking anywhere within the PAL.
* On Client B, click the "[?]" button to the right of the "USERS" header text. Verify that the popup appears and looks better than the old letterbox. Close the popup.
* On Client A, click the "[?]" button to the right of the "ADMIN" tab text. Verify that the popup appears and looks better than the old letterbox. Close the popup.

Thanks for your QAing help! :+1:
